docker/Dockerfile

```diff
@@ -5,9 +5,9 @@ MAINTAINER TensorFlow authors <tensorflow-haskell@googlegroups.com>
 # Installs protoc and the libraries.
 RUN \
     cd /tensorflow && \
-    bazel --batch build -c opt '@protobuf//:protoc' && \
+    bazel --batch build --jobs=7 -c opt '@protobuf//:protoc' && \
```

Comment:
Could you add a sentence to README.md instead? I'm not sanguine about picking a magic number which happens to work now (on your machine with the current set of tools and sources). FYI, I typically run on a 32G machine where this works without any changes.